sys-apps/ucspi-ssl/ucspi-ssl-0.98b.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fixheadtails qmail
+
+DESCRIPTION="Command-line tools for building SSL client-server applications"
+HOMEPAGE="http://www.fehcom.de/ipnet/ucspi-ssl.html"
+SRC_URI="http://www.fehcom.de/ipnet/ucspi-ssl/${P}.tgz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~m68k ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-fbsd"
+IUSE="perl"
+
+DEPEND="
+	>=dev-libs/openssl-0.9.8z_p8:=
```

Comment:
Home page says: "Currently, ucspi-ssl supports OpenSSL 1.0 only (and not 1.1; which provides a significantly changed API to be included for the forthcoming release).". I suppose dependency on exact version is sort of legacy in this ebuild, I proxy-maintain only a couple of last versions and I'm not an author of initial version. I suppose using simple `dev-libs/openssl` should be just fine plus specifically exclude `>=dev-libs/openssl-1.1`. Do you want me to release -r1 with this change or just postpone it for next version bump update?